### PR TITLE
PF compliant buttons: Account settings

### PR DESCRIPTION
**Pull request by damianpm**
**What this PR does / why we need it**:

It standardize links and buttons using patternfly guidelines.

Example:
`<a class="pf-c-button pf-m-link action edit">`
Instead of 
`<a class="action edit">`

Needed for QE automation

**Which issue(s) this PR fixes** 

Fixes https://issues.redhat.com/browse/THREESCALE-6721
Sub task of https://issues.redhat.com/browse/THREESCALE-6691

**IMPORTANT:** 
PR 4 of a series
Merge after:
https://github.com/3scale/porta/pull/2363
https://github.com/3scale/porta/pull/2376
https://github.com/3scale/porta/pull/2384

**PAGES AFFECTED:**

/p/admin/account

/p/admin/account/edit

/p/admin/user/personal_details/edit

/p/admin/user/notification_preferences

/p/admin/account/users/{user_id}/edit

/p/admin/account/authentication_providers/new

/p/admin/account/authentication_providers/{auth_provider_id}

/p/admin/account/authentication_providers/{auth_provider_id}/edit

/p/admin/webhooks/edit

### CODE REVIEW

File: app/views/provider/admin/account/users/_form.html.erb [hunk 1]
```diff
@@ -67,7 +67,7 @@
 <% end %>
 
 <%= form.buttons do %>
-  <%= form.commit_button %>
+  <%= form.button 'Update User', button_html: { class: 'pf-c-button pf-m-primary' } %>
```
Comment (hallelujah) [inline app/views/provider/admin/account/users/_form.html.erb:70]: Same better using locales

File: app/views/provider/admin/accounts/_form.html.erb [hunk 2]
```diff
@@ -5,9 +5,9 @@
 
 <%= form.buttons do %>
   <% if params[:next_step] == 'credit_card' %>
-    <%= form.commit_button 'Save and continue with payment details' %>
+    <%= form.button 'Save and continue with payment details', button_html: { class: 'pf-c-button pf-m-primary' } %>
   <% else %>
-    <%= form.commit_button  %>
+    <%= form.button 'Update Account', button_html: { class: 'pf-c-button pf-m-primary' }  %>
```
Comment (hallelujah) [inline app/views/provider/admin/accounts/_form.html.erb:10]: Same using locales.

File: app/views/provider/admin/user/notification_preferences/_form.html.slim [hunk 3]
```diff
@@ -13,4 +13,5 @@
               hint_method: ->(value) { t(value, scope: :notification_preference_hints) }
 
   = f.buttons do
-    = f.commit_button 'Update Notification Preferences'
+    = f.button 'Update Notification Preferences', button_html: { class: 'pf-c-button pf-m-primary' }
```
Comment (hallelujah) [inline app/views/provider/admin/user/notification_preferences/_form.html.slim:16]: Same using locales

File: app/views/provider/admin/user/personal_details/edit.html.slim [hunk 4]
```diff
@@ -14,7 +14,7 @@ h1 Personal Details
     = form.inputs name: "Provide your current password and update your personal details" do
       = form.input :current_password, required: true
   = form.buttons do
-    = form.commit_button 'Update Details'
+    = form.button 'Update Details', button_html: { class: 'pf-c-button pf-m-primary' }
```
Comment (hallelujah) [inline app/views/provider/admin/user/personal_details/edit.html.slim:17]: Locales :)

File: app/views/provider/admin/webhooks/edit.html.slim [hunk 5]
```diff
@@ -9,4 +9,4 @@ p.help
   = render 'form', form: f
 
   = f.buttons do
-    = f.commit_button label: 'Update webhooks settings'
+    = f.button 'Update webhooks settings',  button_html: { class: 'pf-c-button pf-m-primary' }
```
Comment (hallelujah) [inline app/views/provider/admin/webhooks/edit.html.slim:12]: Locales

File: config/locales/en.yml [hunk 6]
```diff
@@ -1224,7 +1244,7 @@ en:
         update: "%{action} Staging Environment"
       policies:
         update: "Update Policy Chain"
-    labels:
+    l abels:
```
Comment (josemigallas) [inline config/locales/en.yml:1247]:
```suggestion
    labels:
```

**Review by josemigallas — CHANGES_REQUESTED** [review]
LGTM except for one typo

File: config/locales/en.yml (hunk 6 above)
Comment (damianpm) [reply config/locales/en.yml:1247]: Oops

**Review by josemigallas — APPROVED** [review]

**Pull request merged**